From ab85c225e0bf0ea432027715976cea00182f03c5 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Sat, 27 Nov 2010 12:29:14 +0000
Subject: Ensure LIBNOTIFY07 conditional is always set

Also use PKG_CHECK_EXISTS to test if libnotify is recent enough
---
diff --git a/configure.ac b/configure.ac
index dfbce11..4d29d5f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -259,14 +259,13 @@ if test "$have_gtk" = "yes"; then
         AC_SUBST(DBUSGLIB_CFLAGS)
         AC_SUBST(DBUSGLIB_LIBS)
 
-        PKG_CHECK_MODULES(LIBNOTIFY, [ libnotify >= 0.7.0 ],
-          [ libnotify07=yes ],
-          [ PKG_CHECK_MODULES(LIBNOTIFY, [ libnotify ]) ]
-        )
-        AM_CONDITIONAL(LIBNOTIFY07, [ test "$libnotify07" = "yes" ])
+        PKG_CHECK_MODULES(LIBNOTIFY, [ libnotify ])
+        PKG_CHECK_EXISTS([ libnotify >= 0.7.0 ], [ libnotify07=yes ])
+
         AC_SUBST(LIBNOTIFY_CFLAGS)
         AC_SUBST(LIBNOTIFY_LIBS)
 fi
+AM_CONDITIONAL(LIBNOTIFY07, [ test "$libnotify07" = "yes" ])
 
 AM_PROG_VALAC([0.10])
 AC_SUBST(VAPIDIR)
--
cgit v0.8.3-6-g21f6
